platforms/posix/cmake/sitl_target.cmake

```diff
@@ -57,12 +57,15 @@ ExternalProject_Add(mavsdk_tests
 	BUILD_ALWAYS 1
 )
 
+add_subdirectory(${PX4_SOURCE_DIR}/Tools/flightgear_bridge ${PX4_BINARY_DIR}/build_flightgear_bridge EXCLUDE_FROM_ALL)
+
 # create targets for each viewer/model/debugger combination
-set(viewers none jmavsim gazebo)
+set(viewers none jmavsim gazebo flightgear)
 set(debuggers none ide gdb lldb ddd valgrind callgrind)
 set(models none shell
 	if750a iris iris_opt_flow iris_opt_flow_mockup iris_vision iris_rplidar iris_irlock iris_obs_avoid iris_rtps solo typhoon_h480
 	plane plane_cam plane_catapult
+	TF-G1
```

Comment:
Is it at least possible to leave it at ```tf_g1```? Full name of that UAV is [ThunderFly TF-G1](https://github.com/ThunderFly-aerospace/TF-G1).

Reply:
What about `tf-g1` ? I suggest it because in other models in list underscore is used to denote the model variant. But the full name of our model is TF-G1 including the hyphen. (There are much more models in development including TF-G2, TF-G3..  etc.) The letter case does not matter for that.

Reply:
Renamed to tf-g1